Marlin/src/HAL/HAL_LPC1768/HAL_spi.cpp

```diff
@@ -137,28 +124,93 @@
     WRITE(SS_PIN, HIGH);
   }
 #else
-  void spiBegin() {
+
+  #include "../../frameworks/CMSIS/LPC1768/include/lpc17xx_pinsel.h"
```

Comment:
Pretty sure your not using this now with what you said but the framework headers are actually "system" headers so can be included with just <headerfile.h>, library headers in the platform framework can be include this way too.